pkg/apis/hypercloud/v1alpha1/virtualmachinevolumeexport_types.go

```diff
@@ -0,0 +1,88 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// VirtualMachineVolumeExportSpec defines the desired state of VirtualMachineVolumeExport
+type VirtualMachineVolumeExportSpec struct {
+	VirtualMachineVolume VirtualMachineVolumeSource            `json:"virtualMachineVolume"`
+	Destination          VirtualMachineVolumeExportDestination `json:"destination"`
+}
+
+// VirtualMachineVolumeExportDestination defines the destination to export volume
+type VirtualMachineVolumeExportDestination struct {
+	Local *VirtualMachineVolumeExportDestinationLocal `json:"local,omitempty"`
+	S3    *VirtualMachineVolumeExportDestinationS3    `json:"s3,omitempty"`
```

Comment:
S3 is not supported yet, so it is unnecessary

Reply:
Done.